lib/dependency-manager-adapters/npm.js

```diff
@@ -114,7 +114,10 @@ module.exports = CoreObject.extend({
 
     return this.run(cmd, [].concat(['install'], mgrOptions), { cwd: this.cwd }).then(function() {
       if (!adapter.useYarnCommand) {
-        debug('Run npm prune');
+        mgrOptions = mgrOptions.filter(function(opt) {
```

Comment:
Since `prune` has no documented options that cannot be replaced with Env vars, perhaps we shouldn't use mgrOptions at all when running that command. It feels like a bug that we ever did, since the example options were always things for which prune has undocumented behavior.